src/Utils/Strings.php

```diff
@@ -332,10 +332,10 @@ public static function compare($left, $right, $len = NULL)
 	 * @param  string|array
 	 * @return string
 	 */
-	public static function findPrefix($strings)
+	public static function findPrefix(...$strings)
 	{
-		if (!is_array($strings)) {
-			$strings = func_get_args();
+		if (is_array($strings[0])) {
+			$strings = $strings[0];
```

Comment:
IMHO scenario `if (!is_array($var)) $var = func_get_args()` can be deprecated when Nette will require >= 5.6.